scss/hyperdrive-ui.scss

```diff
@@ -0,0 +1,147 @@
+#hyperdrive-ui {
```

Comment:
This was the css that (I believe) was originally copied over from hyperdrive-ui. The wrapping class was a quick fix to avoid any of the styles inside interfere with the new dat-desktop styles (e.g. the global styles for table elements).
I didn't want to delete it yet because I wasn't sure whether you had planned to render any parts of hyperdrive that will need those styles, so using that id would've enabled these styles.

tl;dr: This file can be deleted entirely if you don't need the styles in there :)